doc-files/release-notes-21.md

```diff
@@ -0,0 +1,154 @@
+[JDK-8290092](https://bugs.openjdk.org/browse/JDK-8290092)|Temporary files are kept when call Clipboard.getSystemClipboard().getImage() |graphics
+[JDK-8290866](https://bugs.openjdk.org/browse/JDK-8290866)|Apple Color Emoji turns gray after JavaFX version 18|graphics
+[JDK-8295078](https://bugs.openjdk.org/browse/JDK-8295078)|TextField blurry when inside an TitledPane -> AnchorPane|graphics
+[JDK-8299968](https://bugs.openjdk.org/browse/JDK-8299968)|Second call to Stage.setScene() create sizing issue with uiScale > 1.0|graphics
+[JDK-8300872](https://bugs.openjdk.org/browse/JDK-8300872)|WebView's ColorChooser fails to initialize when running in security context|graphics
+[JDK-8302511](https://bugs.openjdk.org/browse/JDK-8302511)|HitInfo.toString() throws IllegalArgumentException|graphics
+[JDK-8302797](https://bugs.openjdk.org/browse/JDK-8302797)|ArrayIndexOutOfBoundsException in TextRun.getWrapIndex()|graphics
+[JDK-8304831](https://bugs.openjdk.org/browse/JDK-8304831)|TextFlow.hitTest.insertionIndex incorrect with surrogate pairs|graphics
+[JDK-8306708](https://bugs.openjdk.org/browse/JDK-8306708)|Region.layoutInArea uses incorrect snap scale value|graphics
+[JDK-8306990](https://bugs.openjdk.org/browse/JDK-8306990)|Guarantees given by Region's floor and ceiling functions should work for larger values|graphics
+[JDK-8309508](https://bugs.openjdk.org/browse/JDK-8309508)|Possible memory leak in JPEG image loader|graphics
+[JDK-8309935](https://bugs.openjdk.org/browse/JDK-8309935)|Mac - SystemMenuBar, IndexOutOfBoundsException on change|graphics
+[JDK-8313227](https://bugs.openjdk.org/browse/JDK-8313227)|Correct attenuation indicator for removed lights|graphics
+[JDK-8304290](https://bugs.openjdk.org/browse/JDK-8304290)|Some JNI calls made without checking exceptions in media|media
+[JDK-8306328](https://bugs.openjdk.org/browse/JDK-8306328)|Update libFFI to 3.4.4|media
+[JDK-8194704](https://bugs.openjdk.org/browse/JDK-8194704)|Text/TextFlow hitTest() javadoc|scenegraph
+[JDK-8301763](https://bugs.openjdk.org/browse/JDK-8301763)|Adding children to wrong index leaves inconsistent state in Parent#childrenSet|scenegraph
+[JDK-8304933](https://bugs.openjdk.org/browse/JDK-8304933)|BitSet (used for CSS pseudo class states) listener management is incorrect|scenegraph
+[JDK-8304959](https://bugs.openjdk.org/browse/JDK-8304959)|Public API in javafx.css.Match should not return private API class PseudoClassState|scenegraph
+[JDK-8231865](https://bugs.openjdk.org/browse/JDK-8231865)|JFXPanel sends resize event with size 0x0 on HiDPI devices|swing
+[JDK-8242419](https://bugs.openjdk.org/browse/JDK-8242419)|JFXPanel: MouseEvent always reports that Primary button changed state if held|swing
+[JDK-8299977](https://bugs.openjdk.org/browse/JDK-8299977)|Update WebKit to 615.1|web
+[JDK-8301009](https://bugs.openjdk.org/browse/JDK-8301009)|Update libxml2 to 2.10.3|web
+[JDK-8306115](https://bugs.openjdk.org/browse/JDK-8306115)|Update libxml2 to 2.10.4|web
```

Comment:
Do we really need the upgrade to 2.10.3 in the release notes if the release includes 2.10.4?

Reply:
Hmm. Good point. Using somewhat similar reasoning as to why we exclude bugs introduced and fixed in the same release, we can exclude a fix that was added and then superseded in the same release. I'll remove it.